lib/components/layout/layout.tsx

```diff
@@ -36,7 +40,7 @@ export const Layout = forwardRef<HTMLDivElement, LayoutProps>(function Layout(
         ) : null}
         <div
           className={cn("mtfh-layout__main", {
-            "mtfh-layout__main__aside--right": sidePosition === "right",
+            "mtfh-layout__main--right": sidePosition === "right",
```

Comment:
Could we move this check to the root of the component? Then we only have to do it once.

Reply:
```css
.mtfh-layout.mtfh-layout--right {
    .mtfh-layout__main {}
    .mtfh-layout__aside {}
}
```